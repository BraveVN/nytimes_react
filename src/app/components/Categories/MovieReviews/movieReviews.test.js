var MovieReviews = require('./movieReviews');

describe('MovieReviews component', () => {
  test('render the MovieReviews component', () => {
    const wrapper = shallow(<MovieReviews/>);
    expect(wrapper.length).toBe(1);

    var tree = renderer.create(<MovieReviews/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
