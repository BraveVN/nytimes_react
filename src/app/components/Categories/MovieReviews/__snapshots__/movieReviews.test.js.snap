// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieReviews component render the MovieReviews component 1`] = `
<div
  className="category-container"
>
  <h2
    className="title"
  >
    Movie Reviews
  </h2>
  <div
    className="category-content"
  />
</div>
`;
